src/app/Classes/LogViewer.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace Backpack\LogManager\app\Classes;
+use Illuminate\Support\Facades\Storage;
+
+/***
+ * Class LogViewer
+ * @package Backpack\LogManager\app\Classes
+ *
+ * Credits https://github.com/rap2hpoutre/laravel-log-viewer
+ */
+
+class LogViewer
+{
+    /**
+     * @var string file
+     */
+    private static $file;
+
+    private static $levels_classes = [
+        'debug'     => 'info',
+        'info'      => 'info',
+        'notice'    => 'info',
+        'warning'   => 'warning',
+        'error'     => 'danger',
+        'critical'  => 'danger',
+        'alert'     => 'danger',
+        'emergency' => 'danger',
+        'processed' => 'info',
+    ];
+
+    private static $levels_imgs = [
+        'debug'     => 'info',
+        'info'      => 'info',
+        'notice'    => 'info',
+        'warning'   => 'warning',
+        'error'     => 'warning',
+        'critical'  => 'warning',
+        'alert'     => 'warning',
+        'emergency' => 'warning',
+        'processed' => 'info',
+    ];
+
+    /**
+     * Log levels that are used.
+     *
+     * @var array
+     */
+    private static $log_levels = [
+        'emergency',
+        'alert',
+        'critical',
+        'error',
+        'warning',
+        'notice',
+        'info',
+        'debug',
+        'processed',
+    ];
+
+    const MAX_FILE_SIZE = 52428800;
+
+    /**
+     * @param string $file
+     * @throws \Exception
+     */
+    public static function setFile($file)
+    {
+        $file = self::pathToLogFile($file);
+
+        if (app('files')->exists($file)) {
+            self::$file = $file;
+        }
+    }
+
+    /**
+     * @param string $file
+     *
+     * @throws \Exception
+     *
+     * @return string
+     */
+    public static function pathToLogFile($file)
+    {
+        $logsPath = storage_path('logs');
```

Comment:
Should `logs` be hard coded?

Reply:
Yup, I think that's ok, that's where they always live.